src/main/resources/application.yaml

```diff
@@ -4,3 +4,13 @@ server:
 management:
   security:
     enabled: ${MANAGEMENT_SECURITY_ENABLED:true}
+
+redis:
+  host: ${REDIS_HOST}
+  port: ${REDIS_PORT}
+  password: ${REDIS_PASSWORD}
+
+ttlInSeconds: ${TTL_IN_SECONDS:3600}
```

Comment:
It would probably be best to extend this name, so that we would know what it's for. It's a hard one, though. Any ideas?